scripts/test/support.py

```diff
@@ -157,7 +158,9 @@ def run_command(cmd, expected_status=0, stderr=None, expected_err=None):
   out, err = proc.communicate()
   if proc.returncode != expected_status:
     raise Exception(('run_command failed', err))
-  if expected_err is not None and err != expected_err:
+  err_correct = expected_err is None or \
+    (expected_err in err if err_contains else expected_err == err)
```

Comment:
Why not indent this line two extra spaces, instead of disabling that warning project-wide?